templates/1_set_up_toffy.ipynb

```diff
@@ -15,10 +15,10 @@
    "id": "e36293c5-aa89-4029-a3fa-e8ea841bb8b5",
```

Comment:
<p>Key points: Over time, the sensitivity of the detector can change, influencing the output of the detector for the same true amount of signal</p><p> </p><p>The MPH can be used as a metric to evaluate the sensitivity of the detector.</p><p> </p><p>We create a tuning curve to relate MPH to % of max signal to understand the relationship between these two things. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>